Makefile

```diff
@@ -140,42 +140,13 @@ kubectl-config:
 	--user=$(ZQD_K8S_USER)@$(ZQD_TEST_CLUSTER)
 	kubectl config use-context zqtest
 
-helm-install-postgres:
-	helm install postgres charts/postgres \
-	--set postgresql.persistence.enabled=$(PG_PERSIST)
-
-helm-install-recruiter:
-	helm install recruiter charts/zqd \
-	--set AWSRegion=us-east-2 \
-	--set image.repository=$(ZQD_ECR_HOST)/ \
-	--set image.tag=zqd:$(ECR_VERSION) \
-	--set personality=recruiter \
-	--set useCredSecret=false
-
-helm-install-root:
-	helm install root charts/zqd \
-	--set AWSRegion=us-east-2 \
+helm-install:
+	helm install z charts/zservice \
```

Comment:
I'd prefer this name of the app be either "zsrv" or "zservice", but not a hill I willing to die on.